modules/common/src/test/scala/com.snowplowanalytics.snowplow.enrich.common/EtlPipelineSpec.scala

```diff
@@ -44,6 +45,8 @@ class EtlPipelineSpec extends Specification with ValidatedMatchers {
   def is = s2"""
   EtlPipeline should always produce either bad or good row for each event of the payload   $e1
   Processing of events with malformed query string should be supported                     $e2
+  Processing of invalid CollectorPayload (CPFormatViolation bad row) should be supported   $e3
+  Absence of CollectorPayload (None) should be supported                                   $e4
```

Comment:
IIRC we need this because Cloudfront collector supported the format where the first line of file was starting with a comment, like:

```
# timestamp ip ...
2020-02-01 01:32:43 126.10.1.2 ...
```

Need to decide when we're going to drop that officially. Also probably worth to mention it in the test: https://github.com/snowplow/enrich/issues/11